website/content/en/docs/config/disk.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Resize Disk Size
+---
+
+This guide explains how to increase the disk size of a Lima VM running on macOS when you've run out of space, as well as how to edit the disk size using the `limactl` CLI.
+
+## Prerequisites
+1. Sufficient free disk space on your host.
+2. VM type considerations:
+   - For `qemu` VM type: All methods below are supported
+   - For `vz` VM type: Option 1 is supported
+
+## Option 1: Resize Disk Using limactl
+
+Lima supports editing the disk size of an existing instance using the `--disk` flag with the `limactl edit` command.  
+This is the recommended and simplest way to resize your VM disk.
+
+```sh
+limactl edit <vm-name> --disk <new-size>
+```
+Example for 20GB:
+
+```sh
+limactl edit default --disk 20
+```
+
+> **Note:**  
+> - Increasing disk size is supported, but shrinking disks is not recommended.
+> - The instance may need to be stopped before editing disk size.
+
+## Option 2: Manually Resize Lima VM Disk
```

Comment:
This shouldn't be needed

Reply:
Just verifying, 'this' here implies note right?

Reply:
No, I meant that "Option 2" should not be needed.